website/src/app/docs/administer/regen-keys/readme.mdx

```diff
@@ -11,24 +11,17 @@ do so using the same bootstrap scripts that were used when installing Firezone.
 
 ## Regenerate secrets
 
-<Alert
-  color="danger"
-  html={`
-Replacing the <code>DATABASE_ENCRYPTION_KEY</code> will render all encrypted data in the
-database useless. This <strong>will</strong> break your Firezone install unless you are
-starting with an empty database. You have been warned.
-`}
-/>
-
-<Alert
-  color="warning"
-  html={`
-Replacing <code>GUARDIAN_SECRET_KEY</code>, <code>SECRET_KEY_BASE</code>,
-<code>LIVE_VIEW_SIGNING_SALT</code>,
-<code>COOKIE_SIGNING_SALT</code>, and <code>COOKIE_ENCRYPTION_SALT</code>
-will reset all browser sessions and REST API tokens.
-`}
-/>
+<Alert color="danger">
+  Replacing the `DATABASE_ENCRYPTION_KEY` will render all encrypted data in the
+  database useless. This **will** break your Firezone install unless you are
+  starting with an empty database. You have been warned.
+</Alert>
+
+<Alert color="warning">
+  Replacing `GUARDIAN_SECRET_KEY`, `SECRET_KEY_BASE`, `LIVE_VIEW_SIGNING_SALT`,
```

Comment:
So there are three things for the 1.0:
1. `DATABASE_ENCRYPTION_KEY` will render all data unusable, but we don't have it. And I would say unless we build it properly (with an ability to rotate the keys or use Vault for it) it's not really a good idea to build it in: it's a secret that you need to store along with your backups and it never expires, and if you lose it - you lost all the data.
2. Changing `TOKENS_KEY_BASE`, `TOKENS_SALT` will make all existing tokens unusable.
3. `SECRET_KEY_BASE`, `LIVE_VIEW_SIGNING_SALT`, `COOKIE_SIGNING_SALT`, `COOKIE_ENCRYPTION_SALT` will result in all sessions to be expired.

Reply:
Old docs, just being reformatted